CMakeLists.txt

```diff
@@ -626,7 +627,9 @@ psibase_package(
         WASM ${CMAKE_CURRENT_BINARY_DIR}/Webmail.wasm
         SERVER webmail
         DATA ${COMPONENT_BIN_DIR}/${WebmailPlugin_OUTPUT_FILE} /plugin.wasm
+        DATA GLOB ${CMAKE_CURRENT_SOURCE_DIR}/services/user/Webmail/ui/dist/* /
         INIT
+    DEPENDS ${Webmail_js_DEP} ${CMAKE_CURRENT_SOURCE_DIR}/services/user/Webmail/ui/dist/index.html
```

Comment:
`.../index.html` is redundant, because `${Webmail_js_DEP}` already includes it.

Reply:
So should that line simply be `DEPENDS ${Webmail_js_DEP}`?

Reply:
yes